R/defs.bzl

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017 GRAIL, Inc.
+# Copyright Copyright 2018 The Bazel Authors.
```

Comment:
Umm... you have the word Copyright repeated everywhere.

I will merge the PR after this is fixed.